[debug] [2019-09-09T15:54:27.643Z] ----------------------------------------------------------------------
[debug] [2019-09-09T15:54:27.647Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve --only functions --debug
[debug] [2019-09-09T15:54:27.647Z] CLI Version:   7.2.4
[debug] [2019-09-09T15:54:27.647Z] Platform:      darwin
[debug] [2019-09-09T15:54:27.648Z] Node Version:  v10.16.0
[debug] [2019-09-09T15:54:27.649Z] Time:          Mon Sep 09 2019 11:54:27 GMT-0400 (Eastern Daylight Time)
[debug] [2019-09-09T15:54:27.649Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-09-09T15:54:27.667Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-09-09T15:54:27.668Z] > authorizing via signed-in user
[debug] [2019-09-09T15:54:27.669Z] [iam] checking project flashnote-9ed12 for permissions ["firebase.projects.get"]
[debug] [2019-09-09T15:54:27.674Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/flashnote-9ed12:testIamPermissions  

[debug] [2019-09-09T15:54:28.037Z] <<< HTTP RESPONSE 200
[debug] [2019-09-09T15:54:28.044Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/flashnote-9ed12  
 
[debug] [2019-09-09T15:54:28.201Z] <<< HTTP RESPONSE 200
[debug] [2019-09-09T15:54:28.212Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:1147:7)
    at new FunctionsEmulator (/usr/local/lib/node_modules/firebase-tools/lib/emulator/functionsEmulator.js:42:34)
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:39:71)
    at Generator.next (<anonymous>)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:3:12)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:16:16)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
[error] 
[error] Error: An unexpected error has occurred.
